src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -12,6 +15,8 @@
     public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
```

Comment:
Good that you specific the constraints

Reply:
Thanks for noticing my effort, Erwin! Good luck to you on your FXML increments 😚